src/services/localstorage.service.js

```diff
@@ -0,0 +1,25 @@
+export const clearLocalStorage = () => {
```

Comment:
Please, rename the file to `local-storage.service.js`

Reply:
fixed